src/desktop/apps/home/routes.coffee

```diff
@@ -39,6 +39,9 @@ fetchMetaphysicsData = (req, showHeroUnits)->
 @index = (req, res, next) ->
   return if metaphysics.debug req, res, { method: 'post', query: query }
 
+  hideHeroUnits = res.locals.sd.HOME_SEARCH_TEST == 'experiment'
+  res.locals.sd.HIDE_HERO_UNITS = hideHeroUnits  
```

Comment:
I believe this is set when you are in the test group....and you've just loaded the homepage. This is b/c some of the components (search bar), is used in other contexts, and we just want it to have different behavior/appearance when this condition is met (test group and you are on the homepage).

Since `sd.HOME_SEARCH_TEST == 'experiment'` will be true everywhere, on all pages, we wind up using `HIDE_HERO_UNITS` instead (like https://github.com/artsy/force/blob/f8d3bc65b862cc189142744c8a867e3f7a07654b/src/desktop/apps/home/stylesheets/index.styl#L16)

So I think some of the places where you added/checked `testGroup`, as well as `centeredHomepageSearch`, can all be left as-is, checking this value.

This might be my post-vacation haziness tho...

Reply:
Yes, it would work just checking this value via `sd`, but it's better to be explicit about the context in which we want certain presentation changes rather than using `HIDE_HERO_UNITS` everywhere as a proxy for a homepage AB testing context. I prefer being explicit about test groups because it makes it easier to clean up later - it's obviously AB test code.

Reply:
👍 cool sounds good!